docs/local_development.md

```diff
@@ -0,0 +1,27 @@
+# Recommended Local Development and Debugging Workflow
+
+## Overview
+
+We recommend 2 modes of running the broker for development work.
+
+1. Build a local image and deploy from image
+2. Build the broker executable and run it locally connected to a local oc cluster up from [fusor/catasb 'dev' branch](https://github.com/fusor/catasb/tree/dev)
+
+## Build a local image and deploy from image
+
+1. Build the Ansible Service Broker executable with the command: ```make build```
+2. Edit the Makefile to reflect the BROKER_IMAGE_NAME and TAG you want to use for local development
```

Comment:
Do not require editing the Makefile. If we need to change the image name and tag, then those should be passed in as parameters.